client/trino-jdbc/src/test/java/io/trino/jdbc/TestTrinoDriver.java

```diff
@@ -214,8 +214,8 @@ public void testDriverManager()
 
                     assertEquals(rs.getObject(5), true);
                     assertEquals(rs.getObject("_boolean"), true);
-                    assertEquals(rs.getBoolean(5), true);
-                    assertEquals(rs.getBoolean("_boolean"), true);
+                    assertTrue(rs.getBoolean(5));
+                    assertTrue(rs.getBoolean("_boolean"));
```

Comment:
nit: Can this extracted to a seperate commit ?

Reply:
removed